internal/cmd/local/command_kafka_topic_consume.go

```diff
@@ -0,0 +1,163 @@
+package local
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+
+	ckafka "github.com/confluentinc/confluent-kafka-go/kafka"
+
+	"github.com/confluentinc/cli/internal/cmd/kafka"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newConsumeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "consume <topic>",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.topicConsume,
+		Short: "Consume messages from a Kafka topic.",
+		Long:  "Consume messages from a Kafka topic. Configuration and command guide: https://docs.confluent.io/confluent-cli/current/cp-produce-consume.html.\n\nTruncated message headers will be printed if they exist.",
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Consume message from topic "my_topic" from the beginning and with keys printed.`,
+				Code: `confluent local kafka topic consume my_topic --from-beginning --print-key`,
+			},
+		),
+	}
+
+	cmd.Flags().String("group", "", "Consumer group ID.")
+	cmd.Flags().BoolP("from-beginning", "b", false, "Consume from beginning of the topic.")
+	cmd.Flags().Int64("offset", 0, "The offset from the beginning to consume from.")
+	cmd.Flags().Int32("partition", -1, "The partition to consume from.")
+	cmd.Flags().Bool("print-key", false, "Print key of the message.")
+	cmd.Flags().Bool("timestamp", false, "Print message timestamp in milliseconds.")
+	cmd.Flags().String("delimiter", "\t", "The delimiter separating each key and value.")
+	cmd.Flags().StringSlice("config", nil, `A comma-separated list of configuration overrides ("key=value") for the consumer client.`)
+	cmd.Flags().String("config-file", "", "The path to the configuration file (in json or avro format) for the consumer client.")
```

Comment:
```suggestion
	cmd.Flags().String("config-file", "", "The path to the configuration file (in JSON or Avro format) for the consumer client.")

```

Reply:
Can you also add `MarkFilename()` for this flag so autocompletion works a little better?